test/e2e/page-objects/pages/home/non-evm-homepage.ts

```diff
@@ -27,13 +29,27 @@ class NonEvmHomepage extends HomePage {
     }
   }
 
-  protected readonly bridgeButton = '[data-testid="coin-overview-bridge"]';
+  /**
+   * Clicks the bridge button on the non-EVM account homepage.
+   */
+  async clickOnBridgeButton(): Promise<void> {
+    await this.driver.waitForSelector(this.bridgeButton);
+    await this.driver.clickElement(this.bridgeButton);
```

Comment:
we shouldn't wait for a selector and then click on it. This is implicit in the clickElement method
```suggestion
    await this.driver.clickElement(this.bridgeButton);
```

Reply:
Updated